README.md

```diff
@@ -1,134 +1,161 @@
-# eslint-plugin-netguru-ember
+# eslint-plugin-ember
 
-[![NPM version](https://img.shields.io/npm/v/eslint-plugin-netguru-ember.svg?style=flat)](https://npmjs.org/package/eslint-plugin-netguru-ember)
-[![NPM downloads](https://img.shields.io/npm/dm/eslint-plugin-netguru-ember.svg?style=flat)](https://npmjs.org/package/eslint-plugin-netguru-ember)
-[![Circle CI](https://circleci.com/gh/netguru/eslint-plugin-netguru-ember.svg?style=svg&circle-token=58c1b942a91ecd67eed15502a5df51b3d1504f35)](https://circleci.com/gh/netguru/eslint-plugin-netguru-ember)
+[![NPM version](https://img.shields.io/npm/v/eslint-plugin-ember.svg?style=flat)](https://npmjs.org/package/eslint-plugin-ember)
+[![NPM downloads](https://img.shields.io/npm/dm/eslint-plugin-ember.svg?style=flat)](https://npmjs.org/package/eslint-plugin-ember)
+[![Circle CI](https://circleci.com/gh/netguru/eslint-plugin-ember.svg?style=svg&circle-token=58c1b942a91ecd67eed15502a5df51b3d1504f35)](https://circleci.com/gh/netguru/eslint-plugin-ember)
```

Comment:
Any way to make this publicly available? Why include the badge if it's not public? This could be extended into possible move to TravisCI, if necessary.